dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -1220,3 +1220,24 @@ CWWKD1120.cursor.keyword.mismatch.explanation=Keywords such as GROUP BY, HAVING,
  pagination.
 CWWKD1120.cursor.keyword.mismatch.useraction=Rewrite the query to include only \
  the clauses: SELECT, FROM, and WHERE.
+
+CWWKD1121.record.lacks.id=CWWKD1121E: The {0} record cannot be used as an entity \
+ because it lacks a unqiue identifier. One of the record components ({1}) must \
```

Comment:
Instead of `One of the record components ({1})`, change it to `The ({1}) record component`